docs/azure-config.md

```diff
@@ -120,47 +120,40 @@ To integrate Velero with Azure, you must create an Velero-specific [service prin
     AZURE_CLIENT_ID=`az ad sp list --display-name "velero" --query '[0].appId' -o tsv`
     ```
 
-## Credentials and configuration
-
-In the Velero directory (i.e. where you extracted the release tarball), run the following to first set up namespaces, RBAC, and other scaffolding. To run in a custom namespace, make sure that you have edited the YAML file to specify the namespace. See [Run in custom namespace][0].
+Now you need to create a file that contains all the environment variables you just set. The command looks like the following:
 
 ```bash
-kubectl apply -f config/common/00-prereqs.yaml
+cat << EOF  > ./credentials-velero \
```

Comment:
This needs to look like:

```bash
cat > credentials-velero <<EOF
AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID}
AZURE_TENANT_ID=${AZURE_TENANT_ID}
AZURE_CLIENT_ID=${AZURE_CLIENT_ID}
AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET}
AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP}
EOF
```

Reply:
I can change it, though they appear to produce the same result on my machine.

Reply:
Here's what I get if I copy-paste the current version:

```
cat: AZURE_SUBSCRIPTION_ID=: No such file or directory
cat: AZURE_TENANT_ID=: No such file or directory
cat: AZURE_CLIENT_ID=: No such file or directory
cat: AZURE_CLIENT_SECRET=: No such file or directory
cat: AZURE_RESOURCE_GROUP=: No such file or directory
```

Reply:
Double checked on bash and zsh - they produce the same result, though I don't need the trailing `\` on this line.

Reply:
the trailing slashes on all the lines are what cause the issue for me, not the order/spacing of the first line